server/eclipse-project/src/main/java/au/org/democracydevelopers/corla/model/GenerateAssertionsSummary.java

```diff
@@ -0,0 +1,102 @@
+/*
+Democracy Developers IRV extensions to colorado-rla.
+
+@copyright 2024 Colorado Department of State
+
+These IRV extensions are designed to connect to a running instance of the raire 
+service (https://github.com/DemocracyDevelopers/raire-service), in order to 
+generate assertions that can be audited using colorado-rla.
+
+The colorado-rla IRV extensions are free software: you can redistribute it and/or modify it under the terms
+of the GNU Affero General Public License as published by the Free Software Foundation, either
+version 3 of the License, or (at your option) any later version.
+
+The colorado-rla IRV extensions are distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along with
+raire-service. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package au.org.democracydevelopers.corla.model;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+import javax.persistence.*;
+
+import static java.util.Collections.min;
+
+
+/**
+ * RAIRE (raire-java) generates a set of assertions for a given IRV contest, but it also returns
+ * the winner and (possibly) an informative error. These are stored in the database, which
+ * colorado-rla needs to read in order to produce IRV reports. This is read-only table here, with
+ * data identical to the corresponding class in raire-service.
+ */
+@Entity
+@Table(name = "generate_assertions_summary")
+public class GenerateAssertionsSummary {
+
+  /**
+   * Class-wide logger.
+   */
+  private static final Logger LOGGER = LogManager.getLogger(GenerateAssertionsSummary.class);
+
+  /**
+   * ID.
+   */
+  @Id
+  @Column(updatable = false, nullable = false)
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  /**
+   * Version. Used for optimistic locking.
+   */
+  @Version
+  @Column(name = "version", updatable = false, nullable = false)
+  private long version;
+
```

Comment:
If this is meant to be a non-updatable data structure, then it is odd to have non final public attributes. I would make them private and just have getters for the required fields.